web/src/error.rs

```diff
@@ -4,8 +4,8 @@ use axum::http::StatusCode;
 use axum::response::{IntoResponse, Response};
 use serde::Serialize;
 
-use entity_api::error::EntityApiErrorType;
-use entity_api::error::Error as EntityApiError;
+use entity_api::error::EntityApiError;
+use entity_api::error::Error as EntityApiErrorSuper;
```

Comment:
Let's talk through this when we meet later today. I think you've definitely exposed a part of how this is set up that could use more thought. I'm not sure I love adding the `Super` since to me that implies inheritance or nesting. But I also don't have a better option in mind off-hand.

Reply:
Indeed, I just borrowed the `Super` nomenclature from Rust modules which I think makes this fair. My approach also localizes the name distinction to this module only instead of EntityApiErrorType having "Type" always. From my understanding of Rust convention in the community, adding "Type" to the end of a type is not a common practice. It is in C/C++ land, but not Rust.